packages/storefront-lwc/src/modules/commerce/productLineItem/productLineItem.scss

```diff
@@ -0,0 +1,12 @@
+.product-info {
```

Comment:
Im not sure this needs to be nested. the `.remove-btn` should just be in the scope of the `productLineItem` component